src/static/riot/competitions/editor/_phases.tag

```diff
@@ -62,10 +62,10 @@
             Edit phase
         </div>
         <div class="content">
-            <form class="ui form" ref="form">
+            <form selenium="phase-form" class="ui form" ref="form">
                 <div class="field required">
                     <label>Name</label>
-                    <input name="name">
+                    <input selenium="name1" name="name">
```

Comment:
This is extraneous? We should use as few attributes as possible, you should be able to look this up based on the above selenium attribute?

Reply:
Yeah I think that was created for first for another part of the test and then the other selenium attribute was created for another element.